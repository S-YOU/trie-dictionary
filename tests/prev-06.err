nodes: 2
nodes: 4
nodes: 6
nodes: 4
nodes: 2
nodes: 0
