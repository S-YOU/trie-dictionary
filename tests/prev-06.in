insert aaab
prev 0 2 3
prev 0 1 3
delete 1
delete 2
delete 0
