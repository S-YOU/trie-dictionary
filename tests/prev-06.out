word number: 0
word number: 1
word number: 2
deleted: 1
deleted: 2
deleted: 0
